README.md

```diff
@@ -2,34 +2,19 @@ This code is completely based on original https://sourceforge.net/projects/geepr
 
 I wanted some EPROM programmer software which would work on GNU/Linux and I found the above repo that seemed quite abandoned. Wine was not an option.
 
-Following its README instructions I got an error in the first `waf configure` step:
-
-```
-ValueError: invalid mode: 'rU'
-```
-
-I was using python 3.11 but running `waf` with python 2.7 solved the issue.
-
-After that one I got a linker related error:
+11/2025: The build process has been updated so that it runs (at least for me) on newer systems with Python3 and a current version of *maf*, so the old notes about using Python2 are now irrelevant.
```

Comment:
```suggestion
11/2025: The build process has been updated so that it runs (at least for me) on newer systems with Python3 and a current version of *waf*, so the old notes about using Python2 are now irrelevant.
```